packages/foreman/foreman/foreman.spec

```diff
@@ -698,14 +698,14 @@ plugins required for Foreman to work.
 #replace shebangs and binaries in scripts for SCL
 %if 0%{?scl:1}
   # shebangs
-  for f in bin/* script/performance/profiler script/performance/benchmarker script/dynflowd ; do
+  for f in bin/* script/performance/profiler script/performance/benchmarker script/dynflowd; do
```

Comment:
Please keep the diff small and leave this out

Reply:
Sure.